.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+# Description
+Please include a summary of the change and which issue is fixed. Please also include relevant 
```

Comment:
"Please include a summary of the changes and/or which issue is fixed here. ..."

Reply:
Nice catch, should be fixed now.